examples/step-43/step-43.cc

```diff
@@ -802,33 +808,35 @@ namespace Step43
       saturation_matrix.reinit (dsp);
     }
 
-    darcy_solution.reinit (2);
-    darcy_solution.block(0).reinit (n_u);
-    darcy_solution.block(1).reinit (n_p);
+    darcy_index_set.clear();
+    darcy_index_set.resize(2);
```

Comment:
same as before. I suppose the same issue comes up in all of the tutorials below as well, so I won't comment on these any more.